### PR TITLE
#538 update project to 2022.01.xx

### PR DESCRIPTION
**Pull request by MV88**
## Description

This PR updates the project to latest 2022.01.xx

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: project update

## Issue

**What is the current behavior?**

#538 

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

## Other useful information

this is in draft because of #541

### CODE REVIEW

**Comment by offtherailz** [comment]
For the migration guide you are right. They has been changed here but never updated with the fixed versions.

https://github.com/geosolutions-it/MapStore2/commit/1760520f5956823c9b5da23994ffcae66c4ae346

We should document it better. Please open a PR or an issue about the points to fix/improve in migration guidelines.

**Comment by MV88** [comment]
@offtherailz there you go

https://github.com/geosolutions-it/MapStore2/issues/7997

File: web/src/main/resources/proxy.properties [hunk 1]
```diff
@@ -27,4 +27,4 @@ reqtypeWhitelist.capabilities = .*[Gg]et[Cc]apabilities.*
 reqtypeWhitelist.featureinfo = .*[Gg]et[Ff]eature[Ii]nfo.*
 reqtypeWhitelist.csw = .*csw.*
 reqtypeWhitelist.geostore = .*geostore.*
-reqtypeWhitelist.generic = (.*exist.*)|(.*pdf.*)|(.*map.*)|(.*wms.*)|(.*wmts.*)|(.*wfs.*)|(.*ows.*)|(.*spatialreference.*)|(.*wps.*)
+reqtypeWhitelist.generic = (.*exist.*)|(.*pdf.*)|(.*map.*)|(.*wms.*)|(.*wmts.*)|(.*tms.*)|(.*wfs.*)|(.*ows.*)|(.*spatialreference.*)|(.*wps.*)
```
Comment (offtherailz) [inline web/src/main/resources/proxy.properties:30]: You will have to update also this with new types for 3D tiles. See proxy.properties  updated in MapStore when you merge PR from Stefano

**Review by offtherailz — CHANGES_REQUESTED** [review]
Just update `proxy.properties` aligning to latest master, when 3DTiles PR is ported

**Review by offtherailz — APPROVED** [review]

**Pull request merged**